src/collections/_documentation/development/sdk-dev/event-payloads/index.md

```diff
@@ -143,6 +143,8 @@ highly encouraged:
   -   `info`
   -   `debug`
 
+  When this field is set to `fatal` it indicates the application terminated.
```

Comment:
I saw that @bruno-garcia suggested changing from `probably means` to `indicates`.

Since users are eventually in control of levels (*), I'd tend to something less definitive as this depends on the interpretation of parts of code that are out of our control.

I understand that for many SDKs there is no way to assert that the application terminated, because the SDK runs in the same process as the application. E.g. JS Node, JS Browser, Python, PHP, Rust, Go, ...
At most such SDKs may "think" that the application will terminate after the event is reported.

Other than changing icons in the UI, I find it hard to carry any more semantical value from levels. Both because we never explicitly said what each level is supposed to mean and implemented SDKs with that in mind, and because different languages and frameworks carry different semantics (and often choose to have other levels like "critical", "trace", etc)

What do you think?

---

(*) SDKs do set default levels depending on circumstances. E.g. the Go SDK uses `info` for `CaptureMessage`, `error` for `CaptureException` and `fatal` when it recovers from a `panic` (the program may or may not terminate -- depends on the configuration whether to "repanic" or not and whether something else recovers from the panic later).

Reply:
The added sentence is not true for Python at all, and also not the majority case. `fatal` is just another log level.

Reply:
@rhcarvalho I generally agree, but I also think this should be a guideline to be followed, so I think it should be explicit what you need to set to differentiate a hardcrash and a non-hardcrash event.
Right now people don't know how to set nor to filter it.
Let's discuss this over TSC again.

Reply:
What about saying "if this event was created from a log record, this indicates the level with which that message was logged". This delegates the meaning and is still helpful

Reply:
> "if this event was created from a log record, this indicates the level with which that message was logged"

That is better indeed. Unfortunately, it is not always true, as depending on the logging framework the level has to be mapped from some original value to one of those values we accept.

Could be made always true if we accepted arbitrary strings as level, at the moment we don't:

![image](https://user-images.githubusercontent.com/88819/78535389-588f6080-77ec-11ea-9852-a969095b88a4.png)

The invalid value gets overwritten as `level: error`:

![image](https://user-images.githubusercontent.com/88819/78535407-63e28c00-77ec-11ea-8c86-d97c658c2fb2.png)

Reply:
If we don't relax the requirement, we can relax the docs... tried to rewrite Markus' sentence, but then maybe the current docs is just enough?

![image](https://user-images.githubusercontent.com/88819/78535778-069b0a80-77ed-11ea-9f82-123a188b0022.png)

> If this event was created from a log record, the `level` indicates the level with which that message was logged if the logging framework uses the same levels as above.

This rewrite doesn't sound useful to me because it is not assertive.

Reply:
Well oftentimes we can't be assertive, but we can always tell the user where the data came from

Reply:
on Android, we have an `enum` type for levels and not a raw String, so it's up to the Logging integration to pass semantically the right value.